acceptance/tests/facts/sles.rb

```diff
@@ -83,6 +87,7 @@
         kernel_version = '3.0'
       elsif os_version == '12'
         kernel_version = '3.12'
+        kernel_version = '4.4' if os_arch == 'ppc64le'
```

Comment:
What's up with this wildly different kernel version?

Reply:
The Power8 hosts uses SLES 12 SP2, whereas our x86 ones are based on SP1.

Reply:
That is a hilarious kernel bump for a service pack O_O